provider/scripts/k8s/tests/test_offline_store_pandas_runner.py

```diff
@@ -179,6 +179,7 @@ def test_get_blob_credentials(variables, type, request):
     assert credentials == expected_output
 
 
+@pytest.mark.skip("Requires actual connection strings")
```

Comment:
is it possible to still allow these to run but with a certain flag?

Reply:
Changed